lib/template-mixins.js

```diff
@@ -85,34 +85,39 @@ module.exports = function (t, fields, options) {
 
     function optionGroup(key) {
         return {
-            key: key,
-            error: this.errors && this.errors[key],
-            options: _.map(fields[key].options, function (label) {
-                var selected = false, value;
-
-                if (typeof label === 'string') {
-                    value = label;
+            'key': key,
+            'error': this.errors && this.errors[key],
+            'options': _.map(fields[key].options, function (obj) {
+                var selected = false, label, value, toggle;
+
+                if (typeof obj === 'string') {
+                    value = obj;
+                    label = obj;
                 } else {
-                    value = label.value;
-                    label = label.label;
+                    value = obj.value;
+                    label = obj.label;
+                    toggle = obj.toggle;
                 }
 
                 if (this.values && this.values[key] !== undefined) {
                     selected = this.values[key] === value;
                 }
+
                 return {
                     label: t(label),
                     value: value,
-                    selected: selected
+                    selected: selected,
+                    toggle: toggle
                 };
             }, this),
-            display: display(key)
+            'display': display(key)
         };
     }
 
     function checkbox(key, opts) {
         opts = opts || {};
         opts.required = opts.required || false;
+        opts.toggle = fields[key] && fields[key].toggle;
```

Comment:
Because we already have metadata in the fields map for validation of conditional fields, it would possibly be preferable to try to find a way to use that to trigger this behaviour rather than adding an additional point of config to achieve the same thing. 

It may not be possible depending on the markup structure and nature of what's being toggled though. I can't really tell without looking at an implementation. @gavboulton can probably go through it with you on Monday though.